### PR TITLE
feat:added cart page

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Added cart page
image
image
Screenshot (301)
Screenshot (302)

### CODE REVIEW

**Review by suryxks — APPROVED** [review]

File: src/css/cartpage.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	background-color: inherit;
+}
+
+.quantity-item::-webkit-outer-spin-button,
+.quantity-item::-webkit-inner-spin-button {
+	-webkit-appearance: none;
+	margin: 0;
+}
+
+@media (max-width: 1024px) {
```
Comment (manojasarsa) [inline src/css/cartpage.css:54]: I would suggest to use relative units like "em" for media queries. As, when we use pixels to declare our breakpoints, page zooming enacts a horizontal scrollbar and things don't work so great.

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

File: src/css/cartpage.css (hunk 1 above)
Comment (poornima-alugubelly) [reply src/css/cartpage.css:54]: thank you Manoj I'll refactor based on this

**Review by MiheerTamkhane — APPROVED** [review]
Like the practice of utility classes. Good to go I think.

**Pull request merged**